pkg/cmd/server/api/types.go

```diff
@@ -792,6 +792,9 @@ type KubernetesMasterConfig struct {
 	// the server will not start. These values may override other settings in KubernetesMasterConfig which may cause invalid
 	// configurations.
 	ControllerArguments ExtendedArguments
+
+	// KubernetesEtcdClientInfo, if present store kubernetes data in a different etcd instance
+	KubernetesEtcdClientInfo *EtcdConnectionInfo
```

Comment:
update `types_test.go` to set this to a value to ensure we capture any changes to the struct in ansible and docs